src/timeline/route.rs

```diff
@@ -154,6 +154,7 @@ pub fn render_timeline_route(
 #[allow(clippy::too_many_arguments)]
 pub fn render_profile_route(
     id: TimelineId,
+    pubkey: Pubkey,
```

Comment:
we should be passing `&[u8; 32]` or `&Pubkey` here. Since we don't have a Pubkey<'a> type we should probably just do `&[u8; 32]` for flexibility even if we lose type info.